fmatch/matcher.py

```diff
@@ -222,7 +222,7 @@ def burner_cpu_results(self, uuids, namespace, index):
                 "uuid": {
                     "terms": {
                         "field": "uuid.keyword",
-                        "size": self.searchSize
+                        "size": self.search_size
```

Comment:
ah, snake case vs camel case, my bad!

Reply:
No problemo!! was getting an error in pylint, hence had to change.